Source/Base/ASAvailability.h

```diff
@@ -52,11 +52,8 @@
   #define YOGA __has_include(YOGA_HEADER_PATH)
 #endif
 
-// When enabled, use ASTextNode2 for ALL instances of ASTextNode.
-// This includes what ASButtonNode uses internally, as well as all app references to ASTextNode.
-// See ASTextNode+Beta.h declaration of ASTextNodeExperimentOptions for more details.
-#ifndef ASTEXTNODE_EXPERIMENT_GLOBAL_ENABLE
-  #define ASTEXTNODE_EXPERIMENT_GLOBAL_ENABLE 0
+#ifdef ASTEXTNODE_EXPERIMENT_GLOBAL_ENABLE
```

Comment:
Any thoughts on how we can preserve the capability to turn on ASConfiguration options via compile flag? This seems pretty valuable / important, insofar as a JSON config may not be easily plumbed to all environments, like unit tests etc.

I'd say let's land this diff and iterate, although having a workaround for a compiler-driven flag is important for my use case in the short term (let's just say the reasons why are complex...)

Reply:
Good point, I got you covered. I've updated the diff to support this. You can define `AS_FIXED_CONFIG_JSON` as a C-string, and we will implement the `+textureConfiguration` method by parsing that string. Does that solve your use case?